default.nix

```diff
@@ -1,14 +1,40 @@
 {
   niv ? import nix/sources.nix,
-  pkgs ? import niv.nixpkgs { config = { allowUnfree = true; }; }
+  nixpkgs ? niv.nixpkgs,
+  pkgs ? import nixpkgs {}
 }:
+
 let
-  cbsLib = import ./lib { inherit pkgs; };
-  cbspkgs = import ./pkgs { inherit niv pkgs cbsLib; };
-  cbsModules = import ./modules { inherit niv cbspkgs pkgs; };
-in cbspkgs //
-  {
-    lib = cbsLib;
-    nixpkgs = pkgs;
-    modules = cbsModules;
-  }
+  lib = import ./lib { inherit pkgs; };
+  initrdSet = import ./pkgs/initrd-creator/release.nix {
+    inherit pkgs;
+    cbsLib = lib;
+  };
+  kernelSet = import ./pkgs/kernels { inherit pkgs; };
+in pkgs.recurseIntoAttrs rec {
```

Comment:
This is new to me. Does it mean I could do something like `pkgs = niv.cbspkgs-public` and use `pkgs.bender` afterwards? And, in contrast, with the overlay I would say `pkgs.cbspkgs.bender`?

Reply:
Yes.